diff --git a/src/resolveAssetSource.native.ts b/src/resolveAssetSource.native.ts
index 75c2bb7c5a5e0c2195d6ffcc2be3a841c5cec9b7..c9c43ca5457a83e691f7440ae3c31d415f06ccf8 100644
--- a/src/resolveAssetSource.native.ts
+++ b/src/resolveAssetSource.native.ts
@@ -1,3 +1,13 @@
 import resolveAssetSource from 'react-native/Libraries/Image/resolveAssetSource';
 export default resolveAssetSource;
 export * from 'react-native/Libraries/Image/resolveAssetSource'; // eslint-disable-line import/export
+
+import AssetSourceResolver, { ResolvedAssetSource } from './AssetSourceResolver';
+
+let _customSourceTransformer: (resolver: AssetSourceResolver) => ResolvedAssetSource;
+
+export function setCustomSourceTransformer(
+  transformer: (resolver: AssetSourceResolver) => ResolvedAssetSource
+): void {
+  _customSourceTransformer = transformer;
+}
\ No newline at end of file
